pyfr/integrators/dual/pseudo/multip.py

```diff
@@ -93,6 +93,15 @@ def _aux_regidx(iself):
                 def ntotiters(iself):
                     return self.npmgcycles
 
+                if l == self._order:
+                    @property
+                    def stepper_nregs(iself):
+                        return len(iself.stepper_coeffs) - 1
```

Comment:
Is there any chance we know how many stepper coeffs there are when we define the method/class?

Reply:
I can assign this len to a variable in the main multip class init function. Then I can return it instead.

Reply:
Okay, lets see if that is any cleaner

Reply:
Much better!